libs/spark/src/Unstable_IconButton/Unstable_IconButton.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+import { Unstable_IconButton, Unstable_IconButtonProps } from '..';
+import {
+  Unstable_ChevronDown,
+  Unstable_Filter,
+  Unstable_Info,
+} from '../internal';
+import { containFocusIndicator, sparkThemeProvider } from '../../stories';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+interface Sb_Unstable_IconButtonProps
+  extends Omit<Unstable_IconButtonProps, 'tabIndex'> {}
+
+export const Sb_Unstable_IconButton = (props: Sb_Unstable_IconButtonProps) => (
+  <Unstable_IconButton {...props} />
+);
+
+export default {
+  title: '@ps/Unstable_IconButton',
+  component: Sb_Unstable_IconButton,
+  excludeStories: ['Sb_Unstable_IconButton'],
+  parameters: { actions: { argTypesRegex: '^on.*' } },
+  decorators: [containFocusIndicator],
+  argTypes: {
+    children: {
+      control: 'select',
+      options: ['Unstable_ChevronDown', 'Unstable_Filter', 'Unstable_Info'],
+      mapping: {
+        Unstable_ChevronDown: <Unstable_ChevronDown />,
+        Unstable_Filter: <Unstable_Filter />,
+        Unstable_Info: <Unstable_Info />,
+      },
+    },
+  },
+  args: {
+    children: 'Unstable_ChevronDown',
+  },
+} as Meta;
+
+const Template = (args) => <Unstable_IconButton {...args} />;
+
+export const Default: Story = Template.bind({});
+Default.storyName = '(default)';
+
+export const SparkThemeProvider: Story = Template.bind({});
+SparkThemeProvider.decorators = [sparkThemeProvider];
+SparkThemeProvider.storyName = '(SparkThemeProvider)';
+
+const variants: Array<Unstable_IconButtonProps['variant']> = [
+  'primary',
+  'stroked',
+  'ghost',
+];
+const sizes: Array<Unstable_IconButtonProps['size']> = ['medium', 'small'];
```

Comment:
We don't have large?

Reply:
Nope, [figma](https://www.figma.com/file/y6Ca8BdJtQEd3Of1saRieF/components?node-id=5080%3A10852)